Userland/Libraries/LibJS/Tests/builtins/String/String.prototype.replace.js

```diff
@@ -247,4 +247,8 @@ test("UTF-16", () => {
     expect("😀".replace(/\ud83d\ude00/u, "")).toBe("");
 
     expect("".replace("", "😀")).toBe("😀");
+
+    expect("A".replace(/(A)/, "$1")).toBe("A");
+    expect("A".replace(/(A)/, "$10")).toBe("A0");
+    expect("A".replace(/(A)/, "$2")).toBe("$2");
```

Comment:
Nitpick, but these tests have nothing to do with the UTF-16 code unit stuff being tested in this function. Can we move it to it's own `test()` function?

Reply:
Oops, of course!